Automattic/wp-calypso
# App Banner: Display before Editor Welcome Tour

#### Changes proposed in this Pull Request

Right now, the Editor Welcome Tour is showed before the App Banner, and the latter is showed only if the WT has been dismissed and the website is older than 30 minutes.

Now, we want to show the App Banner **before** the Editor Welcome Tour. We can show the Welcome Tour immediately after dismissing the App Banner.

We also want to show the Mobile app banner only at the next visit after the blogger modal has been displayed: pdgK6S-ku-p2

Final priority list:

- blogger modal (if the user is in the Blogger flow)
- app banner (at the next visit to the Editor)
- editor welcome tour (immediately after dismissing the app banner)

#### Testing instructions



1.  Start Calypso, sync ETK (PCYsg-ly5-p2) and sync WBE (PCYsg-l4k-p2)
2. Test the blogger flow, (if you cannot go through the blogger flow, add 'wpcom_signup_complete_show_draft_post_modal' with value 1 in ì browser console -> Application -> Session Storage). So the order of modals will be: blogger modal -> Dismiss -> Refresh -> App Banner -> Dismiss -> Editor Welcome Tour)
 ![Screenshot 2021-12-01 at 14 48 57](https://user-images.githubusercontent.com/52076348/144246007-1d7b8a65-63b3-473f-87bc-0fafdc858771.png)
3. Test that the Blogger Modal is shown. When you dismiss that and refresh you should see the App Banner and immediately after you dismiss that, the Editor Welcome Tour.

![Screen Capture on 2021-12-02 at 11-08-31](https://user-images.githubusercontent.com/52076348/144401545-b2b0cf4d-707f-46a5-ab91-eb2eebbe96f5.gif)

https://d.pr/i/GzEjPr


Related to #58274
Fixes #58274
